docs/src/Row.doc.js

```diff
@@ -0,0 +1,128 @@
+        defaultValue: 'stretch',
+        description:
+          "Aligns a flex container's lines within when there is extra space in the vertical axis, similar to how justify-content aligns individual items within the horizontal axis.",
+      },
+      {
+        name: 'alignItems',
+        type: `"start" | "end" | "center" | "baseline" | "stretch"`,
+        defaultValue: 'center',
+        description:
+          'Defines the default behaviour for how flex items are laid out along the vertical axis on the current line. Think of it as the justify-content version for the vertical axis (perpendicular to the horizontal axis).',
+        href: 'layout',
+      },
+      {
+        name: 'alignSelf',
+        type: `"auto" | "start" | "end" | "center" | "baseline" | "stretch"`,
+        defaultValue: 'stretch',
+        description:
+          'Allows the default alignment (or the one specified by align-items) to be overridden for individual flex items.',
+      },
+      { name: 'fit', type: 'boolean', defaultValue: false },
+      {
+        name: 'flex',
+        type: '"grow" | "shrink" | "none"',
+        defaultValue: 'shrink',
+        description: `Defines how a flex item will be sized. "grow", equivalent to "flex: 1 1 auto", will size the Row relative to its parent, growing and shrinking based on available space. "shrink", equivalent to "flex: 0 1 auto" (the browser default), allows the Row to shrink if compressed but not grow if given extra space. Finally, "none", equivalent to "flex: 0 0 auto", preserves the Row's size based on child content regardless of its container's size.`,
+      },
+      {
+        name: 'gap',
+        type: '0 .. 12',
+        defaultValue: 0,
+        description:
+          'Defines symmetrical horizontal padding to be applied to each child.',
+      },
+      {
+        name: 'justifyContent',
+        type: `"start" | "end" | "center" | "between" | "around" | "evenly"`,
+        defaultValue: 'start',
+        description:
+          'Defines the alignment along the horizontal axis. It helps distribute extra free space left over when either all the flex items on a line are inflexible, or are flexible but have reached their maximum size. It also exerts some control over the alignment of items when they overflow the line.',
+        href: 'layout',
+      },
+      {
+        name: 'wrap',
+        type: 'boolean',
+        defaultValue: false,
+        description: `By default, flex items will all try to fit onto one line. You can change that and allow the items to wrap onto multiple lines, from top to bottom.`,
+      },
+    ]}
+  />
+);
+
+card(
+  <Example
+    description={`
+    With a very limited set of props that only relate to horizontal layout, Row is useful for separating concerns to prevent overloaded Box usage.
+    Compare this example to the first example in the Box docs.
+  `}
+    name="Example: Media object"
+    defaultCode={`
+<Box marginStart={-1} marginEnd={-1}>
```

Comment:
@dangerismycat Can we remove this line? We should no longer need the `<Box marginStart={-1} marginEnd={-1}>` surrounding a row with `gap`

Reply:
yep! good catch!